samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -280,25 +280,23 @@ static int timer_digital_state_cb(uint16_t obj_inst_id,
 static int lwm2m_setup(void)
 {
 	int ret;
-	char *server_url;
+	char server_url[255];
```

Comment:
Hmm, this looks bad as stacks are typically quite small and this eats a lot of that space. Could we avoid this somehow?